src/main/kotlin/com/autonomousapps/internal/advice/filter/FilterSpec.kt

```diff
@@ -114,4 +119,18 @@ internal class FilterSpec(
       universalFilter.predicate(dependency)
     }
   }
+
+  val unusedProcsAdviceFilter: (HasDependency) -> Boolean = { hasDependency ->
+    val dependency = hasDependency.dependency
+    if (anyBehavior is Ignore || unusedProcsBehavior is Ignore) {
+      // If we're ignoring everything or just ignoring all unused procs dependencies, then do that
+      false
+    } else if (anyBehavior.filter.plus(unusedProcsBehavior.filter).contains(dependency.identifier)) {
+      // If we're ignoring some specific dependencies, then do that
+      false
+    } else {
+      // If the dependency is universally ignored, do that
+      universalFilter.predicate(dependency) && facadeFilter.predicate(hasDependency)
```

Comment:
I appreciate that you did this (using `facadeFilter`), but I think we should remove that. For now, at least. I can't think of a scenario where an annotation processor would reasonably be considered a "facade", as I term it. Therefore I'd rather _not_ do this and see if anyone complains and provides a good use-case.